cfme/services/myservice/ui.py

```diff
@@ -148,7 +148,7 @@ def is_displayed(self):
         return (
             self.in_myservices and
             self.myservice.is_opened and
-            self.title.text == 'Set Ownership of Service "{}"'.format(self.context['object'].name))
+            self.title.text == 'Set Ownership for Service')
```

Comment:
This does not appear to uniquely identify the page for the service that ownership is being set for.

I think the correct thing to do in this case, if there's no other unique elements on the page, is to `raise NotImplementedError`.  Checkout the existing implementations to use a consistent string in this instance.